deploy/crds/rhjmc.redhat.com_flightrecorders_crd.yaml

```diff
@@ -368,10 +368,36 @@ spec:
                     description: 'UID of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids'
                     type: string
                 type: object
+              templates:
+                description: Listing of templates available in the target JVM
+                items:
+                  description: TemplateInfo contains metadata for a JFR template
+                  properties:
+                    description:
+                      description: A description of the template and its performance
+                        impact
```

Comment:
I would remove the "and its performance impact" here. The two templates included with recent OpenJDKs might happen to include this as part of the template description, but that isn't in any way guaranteed.